packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -96,6 +98,10 @@ function StepOne({ kycStatus }: StepOneProps) {
           <Text style={styles.description}>{t('linkBankAccountScreen.pending.description')}</Text>
         </View>
       )
+    case KycStatus.NotCreated:
+    case KycStatus.Created:
+    case KycStatus.Pending:
+    case KycStatus.Expired:
```

Comment:
We don't *need* these states here, but I thought it would be better to be explicit. Also, unsure if the fall-through should be considered an anti-pattern or not.